RealJenkinsRuleInit/pom.xml

```diff
@@ -4,16 +4,15 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>4.17</version>
+        <version>4.83</version>
         <relativePath />
     </parent>
     <groupId>io.jenkins.plugins</groupId>
     <artifactId>RealJenkinsRuleInit</artifactId>
     <version>0-SNAPSHOT</version>
     <packaging>hpi</packaging>
     <properties>
-        <jenkins.version>2.249</jenkins.version>
-        <java.level>8</java.level>
```

Comment:
newer version uses java11 and can not set the `java.level` to `11` as it causes `unknown java version 1.11` when compiling.
so updated the parent